yb-voyager/src/srcdb/postgres.go

```diff
@@ -1057,8 +1080,13 @@ The function performs the following checks:
   - Checks if the user has create permission on the database.
   - Checks if the user has ownership over all tables.
 */
-func (pg *PostgreSQL) GetMissingExportDataPermissions(exportType string) ([]string, error) {
+func (pg *PostgreSQL) GetMissingExportDataPermissions(exportType string, finalTableList []sqlname.NameTuple) ([]string, error) {
 	var combinedResult []string
+	var qualifiedQuotedTableList []string
+	for _, table := range finalTableList {
+		qualifiedQuotedTableList = append(qualifiedQuotedTableList, table.ForOutput())
+	}
+	queryTableList := "'" + strings.Join(qualifiedQuotedTableList, "','") + "'"
```

Comment:
use `fmt.Sprintf('%s', strings.Join(qualifiedQuotedTableList, "','"))` for better readability.

Reply:
Done